account_invoice_journal_filter/README.rst

```diff
@@ -0,0 +1,56 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============================
+Account Invioce Journal Filter
+==============================
+
+This module extends the functionality of account invoice to filter by to journal.
```

Comment:
remove "to"
Should be "This module extends functionality of account invoice to filter by journal"